### PR TITLE
feat: add a contact form

### PR DESCRIPTION
**Pull request by aishatpopoola**
This PR does the follwing

- Adds a form spree to the contact form.
- Adds HTML validations to the form inputs.
- ensure that the contact form works perfectly and is submittable and that the submission is sent to a remote email.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -674,13 +693,16 @@ <h3 class="contact_title">location</h3>
...
                 rows="7"
                 class="contact_input"
+                required
+                placeholder="enter your message"
+                maxlength="500"
               ></textarea>
             </div>
             <div>
-              <a href="#" class="button button--flex">
+              <button type="submit" class="button button--flex">
```
Comment (adejam) [inline index.html:702]: please fix the `button--flex` class name. The general convention we are following is either a camelcase or a snake case

**Review by adejam — APPROVED** [review]
#LGTM

please fix the classname, merge and good luck on your next milestone

File: index.html (hunk 1 above)
Comment (aishatpopoola) [reply index.html:702]: ok, it has been fix

**Comment by aishatpopoola** [comment]
I have fixed the issues that needed to be fixed.

**Pull request merged**